packages/dds/matrix/test/testconsumer.ts

```diff
@@ -13,126 +14,75 @@ import { IMatrixConsumer, IMatrixReader, IMatrixProducer } from "@tiny-calc/nano
  * Comparing the state of the TestConsumer with the original IMatrixProducer is a
  * convenient way to vet that the producer is emitting the correct change notifications.
  */
```

Comment:
The only "real" change below is that the TestConsumer reads the cells of newly inserted rows/cols in the row/colChanged callback, which repros the Tablero bug in most of the 'matrix.undo.spec.ts' test cases.

The rest of the delta is opportunistically replacing the custom/internal 2D matrix implementation with the RowMajorMatrix from 'tiny-calc/micro'.